.gitmodules

```diff
@@ -23,3 +23,6 @@
 	path = tests/wasi_testsuite/wasi-common
 	url = https://github.com/WebAssembly/wasi-testsuite.git
 	branch = prod/testsuite-base
+[submodule "wasm-valgrind"]
```

Comment:
I think this is left-over from prior to the import of sources into the main repo -- you should be able to edit `.gitmodules` manually to remove the stanza?